scripts/iota_cache.py

```diff
@@ -0,0 +1,67 @@
+import time
+import sys
+from iota import Iota, Address, ProposedTransaction, Tag, Transaction, TryteString, TransactionTrytes, ProposedBundle
+from six import binary_type, moves as compat, text_type
+
+
+class IotaCache:
+    def __init__(self, uri=None, seed=None):
+        if not uri:
+            self.uri = "http://localhost:14700"
+        if not seed:
+            self.seed = 'EBZYNR9YVFIOAZUPQOLRZXPPPIKRCJ9EJKVCXMYVLMNOCCOPYPJKCWUZNLJZZZZWTMVQUXZFYLVLZXJ9Q'
+        self.api = Iota(self.uri, self.seed)
+
+    def cacheTxnInTangle(self, IPFSAddr, tag):
+        api_response = self.api.get_new_addresses()
+        addy = api_response['addresses'][0]
+        address = binary_type(addy).decode('ascii')
+
+        result = self.api.send_transfer(
+            depth=3,
+            transfers=[
+                ProposedTransaction(
+                    address=Address(address),
+                    value=0,
+                    tag=Tag(tag),
+                    message=TryteString.from_string(IPFSAddr),
+                ),
+            ],
+        )
+        return result
+
+    def getApprovedTxns(self, tag):
+        ret = []
+        transactions = self.api.find_transactions(None, None, [tag], None)
+        if len(transactions['hashes']) == 0:
+            return ret
+        tips = self.api.get_tips()
+        states = self.api.get_inclusion_states(transactions['hashes'], tips['hashes'])
+        i = 0
+        for txn in transactions['hashes']:
+            if states['states'][i] == True:
```

Comment:
if states['states'][i] == True:
May be in this way is recommended:
if states['states'][i] is True:
or
if states['states'][i]: